docs/msbuild/writelinestofile-task.md

```diff
@@ -59,8 +59,33 @@ Writes the paths of the specified items to the specified text file.
     </Target>  
   
 </Project>  
+```
+
+In this example we use a property with `CDATA` to write a text file with multiple lines.
+
+```xml  
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <Target Name="WriteLaunchers" AfterTargets="CopyFilesToOutputDirectory">
+      <PropertyGroup>
+        <LauncherCmd><![CDATA[
```

Comment:
This `CDATA` isn't required; removing it results in the same results (MSBuild allows multiline properties, and even XML embedded in properties).